doc/src/quickstart/index.rst

```diff
@@ -48,6 +48,12 @@ On most unix-like systems you can run:
 
     > make
 
+or, if you will be using :ref:`ODIN II for synthesis<synthesizing_with_odin_ii>`,
+
+.. code-block:: bash
+
+    > make CMAKE_PARAMS="-DWITH_ODIN=on"
```

Comment:
I think that it should be made more clear that Parmys is the default front-end for VTR, but people can build VTR with ODIN by using this build command. Maybe mention that this is required to run "Synthesizing with ODIN II".

Reply:
This should be updated now